### PR TITLE
WIAZ Le-3DliteMkII to TC120PH-2 SP to BB sensor swap

### PR DESCRIPTION
**Pull request by Thomas-Benson**
- Removing the LE-3DliteMkII (I-0641) from WIAZ LC 10
 - Replacing with Trillium Compact 120PH-2 (10438) At new site LC 11
   - New sensor is at 10m depth, undetermined orientation, using zero value for now
   - using coordinates -36.793,175.1339 for new site
 - Stream codes chaning from 10.EH-ENZ to 11.HH-12Z + 11.LH-12Z

### CODE REVIEW

File: network/sites.csv [hunk 1]
```diff
@@ -2237,7 +2237,8 @@ WI14,10,-37.52216,177.18849,25,,WGS84,External GPS Device,2011-06-13T00:00:02Z,2
 WI15,10,-37.52165,177.18668,33,,WGS84,External GPS Device,2011-06-13T00:00:02Z,2011-11-23T00:00:00Z
 WI227,02,-37.52188,177.189942,9,,WGS84,External GPS Device,2019-04-10T01:40:00Z,9999-01-01T00:00:00Z
 WI227,03,-37.52188,177.189942,9,,WGS84,External GPS Device,2019-06-17T23:40:00Z,9999-01-01T00:00:00Z
-WIAZ,10,-36.793,175.1339,255,,WGS84,Internal GPS Clock,2010-07-27T03:15:00Z,9999-01-01T00:00:00Z
+WIAZ,10,-36.793,175.1339,255,,WGS84,Internal GPS Clock,2010-07-27T03:15:00Z,2024-10-09T21:58:00Z
+WIAZ,11,-36.793,175.1339,255,,WGS84,Internal GPS Clock,2024-10-09T21:59:00Z,9999-01-01T00:00:00Z
```
Comment (mnaguit) [inline network/sites.csv:2241]: do we need to change the coordinates a bit (like increase the decimal precision 😄 ) just to distinguish between `LC 10` and `LC 11`? but not to worry, i'm ok with this. 👍

**Review by mnaguit — DISMISSED** [review]
thanks @Thomas-Benson , approving this now noting orientation to be updated later, as needed. 👍

**Comment by Thomas-Benson** [comment]
The new site is  5~6m away, which is within the precision of the current coordinates

File: network/sites.csv (hunk 1 above)
Comment (CBurton90) [reply network/sites.csv:2242]: I'll update channel naming for LC11 now.

Comment (CBurton90) [reply network/sites.csv:2242]: @Thomas-Benson @mnaguit 

channel names updated `Sun Oct 13 20:27:19 UTC 2024`

**Review by CBurton90 — DISMISSED** [review]

**Comment by Thomas-Benson** [comment]
Just need a quick re-review after resolving merge conflicts...

**Review by CBurton90 — APPROVED** [review]

**Review by mnaguit — APPROVED** [review]

**Pull request merged**